foundations/html_css/flexbox/flexbox-growing-and-shrinking.md

```diff
@@ -59,19 +59,19 @@ Here's an example. Note that we've also changed the `flex-basis` for reasons tha
 
 An important implication to notice here is that when you specify `flex-grow` or `flex-shrink`, flex items do not necessarily respect your given values for `width`. In the above example, all 3 divs are given a width of 250px, but when their parent is big enough, they grow to fill it. Likewise, when the parent is too small, the default behavior is for them to shrink to fit. This is not a bug, but it could be confusing behavior if you aren't expecting it.
 
-#### Flex-Basis
+#### Flex-basis
 
 `flex-basis` simply sets the initial size of a flex item, so any sort of `flex-grow`ing or `flex-shrink`ing starts from that baseline size. The shorthand value defaults to `flex-basis: 0%`. The reason we had to change it to `auto` in the `flex-shrink` example is that with the basis set to `0`, those items would ignore the item's width, and everything would shrink evenly. Using `auto` as a flex-basis tells the item to check for a width declaration (`width: 250px`).
 
-> #### Important Note About Flex-Basis:
+> #### Important note About flex-basis:
```

Comment:
```suggestion
> #### Important note about flex-basis:
```

small edit. On code suggestions like these you can `commit suggestion` with the button below it to instantly commit the suggested change if you don't wish to edit it any further and you agree with the change